keyboards/crkbd/keymaps/crkqwes/keymap.c

```diff
@@ -0,0 +1,475 @@
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                       ES_DOT,    ES_1,    ES_2,    ES_3,    ES_0, XXXXXXX,
+    //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                            KC_LGUI, _______,  KC_SPC,     KC_SPC, _______, KC_RALT
+                                        //`--------------------------'  `--------------------------'
+    ),
+
+    [_ADJUST] = LAYOUT_split_3x6_3(
+    //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+          RESET,  RGBRST,RGB_RMOD, XXXXXXX, XXXXXXX, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+        RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+        RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, RGB_SPD, XXXXXXX,                      XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                            KC_LGUI, _______,  KC_SPC,     KC_SPC, _______, KC_RALT
+                                        //`--------------------------'  `--------------------------'
+    )
+};
+
+int RGB_current_mode;
+
+// Setting ADJUST layer RGB back to default
+void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3, uint8_t layer4) {
+  if (IS_LAYER_ON(layer1) && IS_LAYER_ON(layer2) && IS_LAYER_ON(layer3)) {
+    layer_on(layer4);
+  } else {
+    layer_off(layer4);
+  }
+}
+
+void matrix_init_user(void) {
+    #ifdef RGBLIGHT_ENABLE
+      RGB_current_mode = rgblight_config.mode;
+    #endif
+}
+
+// esta configuracion sirve cuando mantenemos precionado una tecla de capa ó block mayus para poder cambiar los colores que Ustedes elijan en hexa se deben poner el color.
+/* void rgb_matrix_indicators_user(void) {
+  #ifdef RGB_MATRIX_ENABLE
+  switch (biton32(layer_state)) {
+    case _RAISE:
+      for (int i = 0; i < DRIVER_LED_TOTAL; i++) {
+          rgb_matrix_set_color(i, 255, 0, 0);
+      }
+      break;
+
+    case _LOWER:
+      for (int i = 0; i < DRIVER_LED_TOTAL; i++) {
+         rgb_matrix_set_color(i, 0, 0, 255);
+      }
+      break;
+
+    default:
+        if (host_keyboard_leds() & (1<<USB_LED_CAPS_LOCK)) {
+                  for (int i = 0; i < DRIVER_LED_TOTAL; i++) {
+          rgb_matrix_set_color(i, 0, 255, 0);
+      }
+       }
+      break;
+  }
+  #endif
+} */
```

Comment:
This doesn't appear to be used, at all?  if that is the case, it should be removed. 
```suggestion
```